lib/mumukit/login/provider/cas.rb

```diff
@@ -0,0 +1,42 @@
+class Mumukit::Login::Provider::Cas < Mumukit::Login::Provider::Base
+  def configure_omniauth!(omniauth)
+    omniauth.provider :cas,
+                      url: cas_config.url,
+                      host: cas_config.host,
+                      ca_path: '.',
+                      disable_ssl_verification: true # FIXME this is big security issue
```

Comment:
We need to remove this line before merging

Reply:
Or at least, parameterize this option with the URL and host